docs/shared-folder.md

```diff
@@ -0,0 +1,36 @@
+# Shared Folder
+
+In HUIT OnDemand, course staff have access to a shared folder for use in their course. Using this shared folder requires use of the terminal, whether through the OnDemand dashboard or through the terminal in the VS Code interface.
+
+The shared folder will be located at `/shared/courseSharedFolders/$CANVAS_COURSE_ID`, where `$CANVAS_COURSE_ID` can be found in your course's Canvas url: `https://canvas.harvard.edu/courses/CANVAS_COURSE_ID`.
```

Comment:
Nice -- this will be helpful for users to quickly locate their shared folder based on their Canvas course.